### PR TITLE
Cosmos DB combined ARM template (was #4489)

### PR DESCRIPTION
**Pull request by WilliamAntonRohm**
### Best Practice Checklist
Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use resourceGroup().location for resource locations
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/bp-checklist.md

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### Changelog

*
*
*

### CODE REVIEW

**Comment by WilliamAntonRohm** [comment]
(replacement for #4489)

**Comment by WilliamAntonRohm** [comment]
@bmoore-msft -- please review, and merge if satisfactory

File: 101-cosmosdb-create-arm-template/metadata.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{ 
+   "itemDisplayName": "Create an Azure Cosmos DB API account", 
+   "description": "This template creates an Azure Cosmos DB API account",
```
Comment (bmoore-msft) [inline 101-cosmosdb-create-arm-template/metadata.json]: Add something here about the different api types - e.g. just grab some of the text from the readme

File: 101-cosmosdb-create-arm-template/azuredeploy.parameters.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "apiType": {
+      "value": "Cassandra"
```
Comment (bmoore-msft) [inline 101-cosmosdb-create-arm-template/azuredeploy.parameters.json]: remove this and use a defaultValue in the template

File: 101-cosmosdb-create-arm-template/metadata.json (hunk 1 above)
Comment (WilliamAntonRohm) [reply 101-cosmosdb-create-arm-template/metadata.json]: Done.

File: 101-cosmosdb-create-arm-template/azuredeploy.parameters.json (hunk 2 above)
Comment (WilliamAntonRohm) [reply 101-cosmosdb-create-arm-template/azuredeploy.parameters.json]: Done

**Comment by WilliamAntonRohm** [comment]
#sign-off

@bmoore-msft -- please review & merge

**Pull request merged**